tests/Rector/ClassMethod/MakeEntitySetterNullabilityInSyncWithPropertyRector/Fixture/many_to_one_default.php.inc

```diff
@@ -7,16 +7,16 @@ use Doctrine\ORM\Mapping as ORM;
 /**
  * @ORM\Entity()
  */
-class Product
+class ManyToOneDefault
 {
     /**
-     * @ORM\ManyToOne(targetEntity="AnotherEntity")
+     * @ORM\ManyToOne(targetEntity="DefaultEntity")
      */
-    private $anotherEntity;
+    private $defaultEntity;
 
-    public function setAnotherEntity(?AnotherEntity $anotherEntity)
+    public function setDefaultEntity(DefaultEntity $defaultEntity)
     {
```

Comment:
Even this is nullable in the database it is business logic acceptable that it stays untouched and the setter is not nullable.

Reply:
What if this entity is used in Symfony forms and it tried to load `null` default values there?

Reply:
Reading is is effecting getters not setter. And would be more somehing which would need to validate on the form that you can not set required false on something which have not a nullable setter. Should not effect the model/entity.

Reply:
Ah, it seems Symfony can handle this: https://github.com/symfony/demo/blob/ecf3f8526c59ebe6cb68b2b79fac5251c9430607/src/Entity/Post.php#L140

All right then :+1: